### ISSUE TITLE
Improper compilation of FDS 5.5.3 with Intel Compiler 13.1.2

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1906]
```
Please complete the following lines...

FDS Version: 5.5.3
SVN Revision Number: 7031
Compile Date: 18/06/2013
Smokeview Version/Revision: 5.6
Operating System: Linux 64 bit
Intel Compiler Version : 13.1.2

Describe details of the issue below:
The issue with FDS 5.5.3 is I could compile it but with errors (presenting only some
part of the error):

../FDS_Source/dump.f90(2290): remark #8291: Recommended relationship between field
width 'W' and the number of fractional digits 'D' in this edit descriptor is 'W>=D+7'.
         WRITE(LU_OUTPUT,'(A,ES8.2)')  '   Viscosity (kg/m/s)   Ambient: ',Y2MU_C(NINT(TMPA))*SPECIES(0)%MW
-------------------------------^
../FDS_Source/main.f90(2313): remark #8291: Recommended relationship between field
width 'W' and the number of fractional digits 'D' in this edit descriptor is 'W>=D+7'.
            WRITE(LU_ERR,'(A,I3,A,I3,A,E12.6)') 'Volume Flow Error, Meshes ',NM,' and
',NOM,' = ',ERROR
----------------------------------------^
Though the executable was created after compilation, it is understood that it results
in unsuccessful execution of fds case file.

When run in serial, the following error occured:
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source        

fds5_mpi_intel_li  0000000000918AAD  Unknown               Unknown  Unknown
fds5_mpi_intel_li  00000000009F37B4  Unknown               Unknown  Unknown
fds5_mpi_intel_li  000000000040428C  Unknown               Unknown  Unknown
libc.so.6          0000003D9EC1ECDD  Unknown               Unknown  Unknown
fds5_mpi_intel_li  0000000000404189  Unknown               Unknown  Unknown

In parallel, though the number of processes were given same, it kept on showing the
error that MPI_PROCESS greater than total number of processes.

Please find the attached zip file containing input case file for both serial and parallel
cases.
```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-19 07:00:27

<hr>
- _Attachment: [FDS 5.5.3_r7031_Issue.rar](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1941/comment-0/FDS 5.5.3_r7031_Issue.rar)_

**Comment by gforney** [comment]
```
Correction :

Compile Date : Fri, 29 Oct 2010
```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-19 11:01:34

**Comment by gforney** [comment]
```
The case ran for me using

 Intel(R) Fortran 13.1-2155

Have you ever ran the MPI version before? It sounds like the parallel version is not
working.
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-19 19:02:46

**Comment by gforney** [comment]
```
Currently, we are using openMPI.

Version Details:
Open MPI : 1.6.3

```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-20 07:36:34

**Comment by gforney** [comment]
```
Have you ever successfully run an MPI job on your current machine? Have you ever successfully
compiled and run and MPI job?
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-20 13:29:59

**Comment by gforney** [comment]
```
Issue 1942 has been merged into this issue.
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-20 13:32:28

**Comment by gforney** [comment]
```
Yes. I have run the older versions of FDS using MPI as well.
```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-20 14:13:33

**Comment by gforney** [comment]
```
But have you ever successfully compiled the FDS source code (MPI version), and then
successfully run a case?
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-20 14:14:49

**Comment by gforney** [comment]
```
Yes, We could successfully compile FDS 5.5.0(MPI) with GNU compiler. But, faced some
issues with HRR not matching. As there is no support for older versions, we shifted
to FDS 5.5.3 and FDS 6 but could not compile it with GNU compilers. We were asked to
use Intel compilers only. After getting Intel compiler, tried FDS 5.5.3(MPI) which
resulted in errors and FDS 6(MPI) which we compiled it successfully.

Refer to the below links from discussions for more details :

https://groups.google.com/forum/?fromgroups#!topic/fds-smv/pVEi1RWvBuQ

https://groups.google.com/forum/?fromgroups#!topic/fds-smv/dKxF8Y_jSdw
```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-20 14:56:56

**Comment by gforney** [comment]
```
If you can successfully compile and run both the serial and parallel versions of FDS
6, then we will close this issue. I am not sure why you cannot compile FDS 5.5.3. I
can only suggest that you dial back the entire FDS-SMV repository to 7031 and try again.
But I cannot worry too much about compiling older versions. We are focused on making
FDS 6 as best we can.
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-20 17:42:12

**Comment by gforney** [comment]
```
I just tried to give you the whole picture of it and it seems you completely missed
my point. If I could successfully compile and run FDS 6, then I would not worry about
older versions. In the other issue 1942., I clearly mentioned I COULD NOT RUN THE PARALLEL
VERSION OF FDS 6 though I did not face any issues while compiling it. In this issue,
I mentioned that I could not even compile FDS 5.5.3.

As support is offered for both these versions, I was just hoping that you would resolve
either one of these issues.

```

Original issue reported on code.google.com by `srisaimeher.k` on 2013-06-21 04:39:41

**Comment by gforney** [comment]
```
Our support extends only so far. I cannot know exactly how you have configured your
OS, and you have not provided enough information about the specific problem to allow
me to help. 
```

Original issue reported on code.google.com by `mcgratta` on 2013-06-21 12:21:22
